packages/api/src/hooks/useIsEuRegion.ts

```diff
@@ -35,9 +35,21 @@ const useIsEuRegion = () => {
         return is_financial_maltainvest || is_eu_based_on_shortcodes || is_eu_based_on_residence;
     }, [landing_company]);
 
+    // New method to test, if this works will remove the legacy method above
+    const isEU = useMemo(() => {
+        if (!landing_company) return;
+
+        const { gaming_company, financial_company } = landing_company;
+
+        const isEuRegion = !gaming_company && financial_company?.shortcode === 'maltainvest';
```

Comment:
@thisyahlen-deriv this  condition just covers the EU countries, the DIEL EU countries( South Africa, ....) fallow this condition 
`gaming_company.shortcode === 'svg' && financial_company?.shortcode === 'maltainvest'`
so we need to differentiate between EU and DIEL EU

Reply:
I have another hook for that

Reply:
This one should only be for EU countries

Reply:
in a different PR?

Reply:
yeaps, this PR is to make my other PR much more easier to handle since i can differentiate after this PR between EU and Non-EU content.